collect_app/src/main/java/org/odk/collect/android/activities/WebViewActivity.java

```diff
@@ -0,0 +1,89 @@
+package org.odk.collect.android.activities;
```

Comment:
Please add a license header with your name in it https://github.com/opendatakit/collect/blob/master/LICENSE.md#appendix-how-to-apply-the-apache-license-to-your-work